### PR TITLE
EDU-5998 Feedback - Update VTEX - Catalog API.json

### PR DESCRIPTION
**Pull request by julia-rabello**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [ ] No, but I am going to.

[EDU-5998](https://vtex-dev.atlassian.net/browse/EDU-5998)

### CODE REVIEW

File: VTEX - Catalog API.json [hunk 1]
```diff
@@ -17036,7 +17036,7 @@
                 }
             }
         },
-        "/api/catalog/pvt/specification/nonstructured/{skuId}": {
+        "/api/catalog/pvt/specification/nonstructured/skuId={skuId}": {
```
Comment (karenkrieger) [inline VTEX - Catalog API.json:17039]: Se for uma query, ela não fica no path, é só adicionar nos parms como query e required

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: VTEX - Catalog API.json (hunk 1 above)
Comment (karenkrieger) [reply VTEX - Catalog API.json:17039]: Esse `skuId=` já indica que é uma query, se fosse um path param seria `{skuId}`

Comment (julia-rabello) [reply VTEX - Catalog API.json:17039]: Te chamei no Slack

Comment (julia-rabello) [reply VTEX - Catalog API.json:17039]: Ajustado! @karenkrieger

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**